<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_data</class>
 <widget class="QWidget" name="settings_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Data Selector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Weather Properties to Display:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>763</width>
         <height>963</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="scrollLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxTemperature">
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxDewpoint">
          <property name="text">
           <string>Dewpoint</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxMaxTemperature">
          <property name="text">
           <string>Max Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxMinTemperature">
          <property name="text">
           <string>Min Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRelativeHumidity">
          <property name="text">
           <string>Relative Humidity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxApparentTemperature">
          <property name="text">
           <string>Apparent Temperature (Feels Like)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWetBulbGlobeTemperature">
          <property name="text">
           <string>Wet Bulb Globe Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxHeatIndex">
          <property name="text">
           <string>Heat Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWindChill">
          <property name="text">
           <string>Wind Chill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSkyCover">
          <property name="text">
           <string>Sky Cover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWindDirection">
          <property name="text">
           <string>Wind Direction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWindSpeed">
          <property name="text">
           <string>Wind Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWindGust">
          <property name="text">
           <string>Wind Gust</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWeather">
          <property name="text">
           <string>Weather Conditions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxHazards">
          <property name="text">
           <string>Hazards</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxProbabilityOfPrecipitation">
          <property name="text">
           <string>Probability of Precipitation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxQuantitativePrecipitation">
          <property name="text">
           <string>Quantitative Precipitation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxIceAccumulation">
          <property name="text">
           <string>Ice Accumulation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSnowfallAmount">
          <property name="text">
           <string>Snowfall Amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSnowLevel">
          <property name="text">
           <string>Snow Level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxCeilingHeight">
          <property name="text">
           <string>Ceiling Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxVisibility">
          <property name="text">
           <string>Visibility</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxTransportWindSpeed">
          <property name="text">
           <string>Transport Wind Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxTransportWindDirection">
          <property name="text">
           <string>Transport Wind Direction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxMixingHeight">
          <property name="text">
           <string>Mixing Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxHainesIndex">
          <property name="text">
           <string>Haines Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxLightningActivityLevel">
          <property name="text">
           <string>Lightning Activity Level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxTwentyFootWindSpeed">
          <property name="text">
           <string>20-Foot Wind Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxTwentyFootWindDirection">
          <property name="text">
           <string>20-Foot Wind Direction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWaveHeight">
          <property name="text">
           <string>Wave Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWavePeriod">
          <property name="text">
           <string>Wave Period</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWaveDirection">
          <property name="text">
           <string>Wave Direction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxProbabilityOfThunder">
          <property name="text">
           <string>Probability of Thunder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxAtmosphericDispersionIndex">
          <property name="text">
           <string>Atmospheric Dispersion Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxLowVisibilityOccurrenceRiskIndex">
          <property name="text">
           <string>Low Visibility Occurrence Risk Index</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="selectAllButton">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deselectAllButton">
        <property name="text">
         <string>Deselect All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply Selection</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
